upstream-community-operators/rook-ceph/rook-ceph.package.yaml

```diff
@@ -1,4 +1,4 @@
-packageName: rook-ceph
+packageName: Rook-Ceph
```

Comment:
no need to have this be capitalized - this is not the display name

```suggestion
packageName: rook-ceph
```

Reply:
But for consistency can we keep it capitalized?

Reply:
no, this is an system-internal name and changing it would result in another Operator bundle being presented.

Reply:
Yes and the CI wasn't happy anyways.